modules/test-structure/validate_struct.go

```diff
@@ -69,6 +82,28 @@ func NewValidationOptions(rootDir string, includeDirs, excludeDirs []string) (*V
 	return vo, nil
 }
 
+// NewValidationOptions returns a ValidationOptions struct, with override-able sane defaults, configured to find
+// and process all directories containing .tf files
+func NewValidationOptions(rootDir string, includeDirs, excludeDirs []string) (*ValidationOptions, error) {
```

Comment:
By splitting `NewValidationOptions` into two separate functions, one that configures `ValidationOptions` with a `FileType` of `TF` for Terraform, and one that configures `ValidationOptions` with a `FileType` of `TG` for Terragrunt, we can a backward incompatible release for the tests already calling the `NewValidationOptions` function.